src/main/java/ch/uzh/ifi/hase/soprafs22/controller/LobbyController.java

```diff
@@ -127,4 +131,22 @@ public List<LobbyGetDTO> getLobby() {
         return lobbiesGetDTOs;
 
     }
+
+    @PostMapping("{API_VERSION}/game/lobby/{id}/player")
+    @ResponseStatus(HttpStatus.OK)
+    @ResponseBody
+    public Map<String, Object> addPlayer(@RequestHeader("token") String token, @PathVariable Long id, @RequestBody PlayerPostDTO playerPostDTO) {
```

Comment:
Sorry, but could you create a PlayerGetDTO instance from the newPlayer and return that instead of this Map<String, Object>?

Reply:
sure!

Reply:
thanks for the review! on other comments including